woocommerce/woocommerce-android
# [HACK Week: Wear app] Update WooCommerce Wear complications icons

Summary
==========
Updates the Wear complications default icon to match the new design guideline for the Woo brand.

This PR contains no unit test since this is a direct icon swap.

source: pe5pgL-5g7#comment-4269

Screen Capture
==========
https://github.com/user-attachments/assets/59c09e03-3457-48b4-bd04-81a8b916ce9d



How to Test
==========
1. Install the Woo app and go to the Wear watch face screen
2. Start the Watch face selection, select one with proper support for Complications with icons
3. Select the Woo `App Shortcut` complication, verify the item in the list follows the new icon
4. Verify the complication preview is also following the same new icon
5. Confirm the complication selection, verify that the complication correctly starts the app and is also following the same new icon

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
